tools/reproduce_comparison/Jenkinsfile

```diff
@@ -37,7 +38,7 @@ pipeline {
                         if (params.platform != 'linux') {
                             "prep${platform}" ('url1Dir1', 'url1Dir2')
                         }
-                        def retVal = sh returnStatus: true, script: 'diff -q -r url1Dir1 url1Dir2'
+                        def retVal = sh returnStatus: true, script: 'diff -q -r url1Dir1 url1Dir2 --exclude=${params.excludeFile}'
```

Comment:
We do not need to introduce input as Jenkins parameters.
just make a hardcoded ArrayList is enough in this case. Becuse this is only running on linux, in other two OSs it is not clear how it should be done.
Until we want to create differnet jobs per different OS might be useful to take in user input in different cases.

One more thing, we need to make it work with multiple files.
At the moment, the only one is classes_nocoops.jsa but likely more will be added into the list.

And the file to be excluded is called `classes_nocoops.jsa` not `classes_nocoops`

Reply:
@zdtsw `classes_nocoops` is a pattern, so all files matching that pattern will be excluded. Diff doesn't support taking exclude input as a list of arrays so if we have to work with multiple files, we will need to append `--exclude` multiple times.

Reply:
We only want to exclude `classes_nocoops.jsa` if there is another one match `classes_nocoops` we do not want to exclude it unless we are certain. Otherwise the requirement for comparison is inaccurate. 

As for the description of this issue is to have a list of it: originally two files `release` and `classes_nocoops.jsa` both should be excluded, then, `release` is removed so the title was updated with one `classes_nocoops.jsa`
Suggestion from comment is just an example for "good first issue" to start with, not as instruction. 
You need to add a loop and figure out how to handle say more than one file in this case.

Reply:
Understood, will add support to provide input by `,` delimiter & then iterate over the strings.

Reply:
@zdtsw @karianna I have made the requested change!